src/tests/ftest/pool/destroy_rebuild.yaml

```diff
@@ -1,28 +1,24 @@
 # change host names to your reserved nodes, the
 # required quantity is indicated by the placeholders
+timeout: 120
 hosts:
-  test_machines:
+  test_servers:
     - boro-A
     - boro-B
     - boro-C
     - boro-D
     - boro-E
     - boro-F
+  test_clients:
```

Comment:
Do we really need a separate client node? As it's just a single node I assume there is going to be single client process at a time. Hence we can actually use avocado's client agent. The node from which the tests are launched acts like the avocado's client agent. We don't need to explicitly specify separate client node in that case.

Reply:
Based upon the similar conversation in #703, can this be resolved?